### PR TITLE
chore(containers): register all components and containers

### PR DESCRIPTION
**Pull request by jmfrancois**
**What is the problem this PR is trying to solve?**

Each container are registred one by one; All components in @talend/react-components are not.
Bootstrap component like NavItem can't be used by inject.

**What is the chosen solution to this problem?**

Just register all the components and containers available in the default index.
bootstrap component are not design to be connected in cmf. They don't have Inject.
Add a wrap function to connect them and adding only one slot: 'children'

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/containers/src/register.js [hunk 1]
```diff
@@ -1,66 +1,26 @@
...
-	api.component.register('ObjectViewer', ObjectViewer);
-	api.component.register('Redirect', Redirect);
-	api.component.register('ShortcutManager', ShortcutManager);
-	api.component.register('SelectObject', SelectObject);
-	api.component.register('SidePanel', SidePanel);
-	api.component.register('SubHeaderBar', SubHeaderBar);
-	api.component.register('TooltipTrigger', TooltipTrigger);
-	api.component.register('TreeView', TreeView);
-	api.component.register('Typeahead', Typeahead);
+	const mycontainers = omit(containers, ['actionAPI']);
```
Comment (talend-glorieux) [inline packages/containers/src/register.js]: Replace `my` with something meaningful.

File: packages/containers/src/register.js [hunk 2]
```diff
@@ -1,66 +1,26 @@
...
+	const toOmit = Object.keys(mycontainers);
+	const connected = {};
+	// connect our component which provide a great api
+	Object.keys(omit(components, toOmit)).forEach(key => {
+		if (components[key]) {
+			connected[key] = cmfConnect({})(components[key]);
+		}
+	});
```
Comment (talend-glorieux) [inline packages/containers/src/register.js]: Use a `map` instead of `forEach`. It avoids creating a contant which isn't one.

Comment (jmfrancois) [reply packages/containers/src/register.js]: `map` is not appropriate to build an object. You mean `reduce` ?

File: output/containers.eslint.txt [hunk 3]
```diff
@@ -11,13 +11,18 @@ The react/require-extension rule is deprecated. Please use the import/extensions
...
 /home/travis/build/Talend/ui/packages/containers/src/renderers.js
   5:33  error  Unexpected parentheses around single function argument  arrow-parens
 
 /home/travis/build/Talend/ui/packages/containers/src/SidePanel/SidePanel.connect.js
   3:1  error  'react-router' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
 
-✖ 5 problems (5 errors, 0 warnings)
+/home/travis/build/Talend/ui/packages/containers/src/wrap.test.js
+  7:20  error  'onClick' is missing in props validation   react/prop-types
+  7:29  error  'children' is missing in props validation  react/prop-types
```
Comment (talend-glorieux) [inline output/containers.eslint.txt]: To fix.

**Pull request merged**